### DISCUSSION TITLE
加载更多案例?

### DISCUSSION THREAD

**Discussion by Jairwin-L** [alovajs/alova#540] [Q&A]
列表数据第一次渲染的时候, 可以全局loading, 然后点击加载更多的时候, 可以理解为“加载更多”变成“加载中……”, 但是全局loading就不需要了, 只需要局部loading, 这个使用场景有文档? 我只看到一个append, 但这个好像不太符合?

**Comment by JOU-amjs** [comment]
添加一个判断，data为空的时候为全局loading

**Reply by Jairwin-L** [reply]
好的, 后面我看看
